scripts/nexus/setup-nexus.sh

```diff
@@ -319,44 +319,45 @@ function setup-nexus-server() {
             repo_name="${name}-$(basename "$directory")"
             if ! nexus-create-repo "$repo_name" yum; then
                 echo >&2 "Failed to create repo: $directory"
+                exit 1
```

Comment:
The bash function shouldn't exit.
Instead of adding a dozen exit statements, just wrap the call to setup-nexus-server with an if statement, where it can print and exit immediately.

Reply:
good catch!